src/WebJobs.Script.WebHost/FunctionsSyncService.cs

```diff
@@ -49,18 +56,32 @@ internal bool ShouldSyncTriggers
             }
         }
 
+        internal bool ShouldStartTimer
+        {
+            get
+            {
+                // only want to do background sync triggers when NOT
+                // in standby mode and not running locally
+                return !_environment.IsCoreToolsEnvironment() &&
+                    !_webHostEnvironment.InStandbyMode && _environment.IsContainerReady();
```

Comment:
Move this logic to FunctionsSyncManager? That will centralize the conditions when you call synctriggers

Reply:
Done